src/ni/vsbuild/BuildConfiguration.groovy

```diff
@@ -77,4 +79,31 @@ class BuildConfiguration implements Serializable {
 
       return list
    }
+
+   private static def replaceTags(def script, def jsonItem, def lvVersion) {
+      if(jsonItem instanceof java.lang.String ||
+         jsonItem instanceof org.codehaus.groovy.runtime.GStringImpl) {
```

Comment:
Nitpick: take the highest class in the [inheritance hierarchy](http://docs.groovy-lang.org/docs/groovy-2.3.3/html/api/org/codehaus/groovy/runtime/GStringImpl.html) that meets the behavior you're looking for: 

```
      if(jsonItem instanceof java.lang.String ||
         jsonItem instanceof groovy.lang.GString) {
```